src/crud.js

```diff
@@ -0,0 +1,34 @@
+export const add = (todos, item) => {
+  todos.push({
+    description: item,
+    index: todos.length + 1,
+    completed: false,
+  });
+  window.localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+export const remove = (todos, id) => {
+  const newTodos = todos.filter((el) => el.index !== id);
+  newTodos.sort((a, b) => a.index - b.index);
+  for (let i = 1; i <= newTodos.length; i += 1) {
+    newTodos[i - 1].index = i;
+  }
+  console.log(newTodos);
```

Comment:
Optional: Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.